frontend/app/client-dashboard/projects/page.js

```diff
@@ -1,30 +1,103 @@
+"use client";
+import { useState, useEffect } from "react";
 import ProjectClient from "@/app/_components/client-dashboard-components/ProjectClient";
 
-export const metadata = {
-  title: "projects",
-};
-const URL =
-  "https://wflance-production.up.railway.app/api/pj";
-
-export default async function Page() {
-  const response = await fetch(URL);
-  const projects = await response.json();
-  console.log(projects);
-  if (!response.ok) {
-    return (
-      <div>
-        Error fetching projects: {response.statusText}
-      </div>
-    );
-  }
+// export const metadata = {
```

Comment:
why is this left here? :)

Reply:
because i have to handle it later, we need this part in this page, now beacuse i use "use client" it doesn't work